supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -508,6 +472,17 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 
 *See also*: xref:red-hat-enterprise-linux-openstack-platform[Red Hat Enterprise Linux OpenStack Platform]
 
+[[red-hat-process-automation-manager]]
+==== image:images/yes.png[yes] Red Hat Process Automation Manager (noun)
+*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Bu using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies. The platform is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.
```

Comment:
```suggestion
*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Bu using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies.
```
The last sentence is pretty marketing-focused, and not needed in an SSG glossary entry.